src/Roslyn.Diagnostics.Analyzers/UnitTests/DeclarePublicAPIAnalyzerTests.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Test.Utilities;
+using Test.Utilities.Legacy;
```

Comment:
We are not updating all tests to use the new test framework? Was there a specific criteria that you used to decide which tests to port?

Reply:
> Was there a specific criteria that you used to decide which tests to port?

I wanted to give a sampling for a few projects without spending the time to do a complete port until I at least get some initial feedback. There are a lot of tests in this project. 😆